icu4c/source/test/intltest/intltest.h

```diff
@@ -166,30 +166,27 @@ class IntlTest : public TestLog {
     virtual void logln( void );
 
     /**
-     * Replaces isICUVersionAtLeast and isICUVersionBefore
-     * log that an issue is known.
+     * Log that an issue is known. Can be called multiple times.
```

Comment:
optional: i prefer descriptive "Logs that..." rather than imperative "Log that..."